Analytics/Classes/Integrations/SEGIntegrationsManager.m

```diff
@@ -26,9 +26,9 @@
 #import "SEGAliasPayload.h"
 
 NSString *SEGAnalyticsIntegrationDidStart = @"io.segment.analytics.integration.did.start";
-static NSString *const SEGAnonymousIdKey = @"SEGAnonymousId";
-static NSString *const kSEGAnonymousIdFilename = @"segment.anonymousId";
-static NSString *const SEGCachedSettingsKey = @"analytics.settings.v2.plist";
+NSString *const SEGAnonymousIdKey = @"SEGAnonymousId";
+NSString *const kSEGAnonymousIdFilename = @"segment.anonymousId";
+NSString *const kSEGCachedSettingsFilename = @"analytics.settings.v2.plist";
```

Comment:
I also changed this `const` name in order to follow the convention established by other filenames/constants whin the project.